lib/blockchain.go

```diff
@@ -1950,16 +2006,29 @@ func CheckTransactionSanity(txn *MsgDeSoTxn, blockHeight uint32, params *DeSoPar
 	return nil
 }
 
-func GetReorgBlocks(tip *BlockNode, newNode *BlockNode) (_commonAncestor *BlockNode, _detachNodes []*BlockNode, _attachNodes []*BlockNode) {
+func (bc *Blockchain) GetReorgBlocks(tip *BlockNode, newNode *BlockNode) (
```

Comment:
I feel like this shouldn't ever be called for txindex, though, FWIW so not sure we need to optimize it.

Reply:
This is still needed for some old PoW tests. so I'm just going to leave it unoptimized since we won't be hitting PoW reorgs in practice.